android/build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 32
+    compileSdkVersion 33
```

Comment:
Why do we need to increase this version?

Reply:
Sorry, it was shot in the dark, rollbacked it, I tried to fix Android tests, they are passing locally, but not in github-actions, and unfortunately I can find any way how to turn on human readable logs, they are generating 100K lines of logs it is absolutely impossible to understand what is wrong